local-cli/generator-macos/templates/macos/Podfile

```diff
@@ -3,41 +3,30 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 
 target 'HelloWorld-macOS' do
   platform :macos, '10.15'
-  config = use_native_modules!
+  use_native_modules!
   use_react_native!(
-    :path => config[:reactNativePath],
-    # to enable hermes on iOS, change `false` to `true` and then install pods
-    :hermes_enabled => false
+    :path => '../node_modules/react-native-macos',
+    # :hermes_enabled => true
   )
 
-  # Enables Flipper.
-  #
-  # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable the next line.
-  # use_flipper!()
-
   # Pods specifically for macOS target
 end
 
 target 'HelloWorld-iOS' do
-  platform :ios, '11.0'
-  config = use_native_modules!
-  use_react_native!(
-    :path => config[:reactNativePath],
-    # to enable hermes on iOS, change `false` to `true` and then install pods
-    :hermes_enabled => false
-  )
+  platform :ios, '11'
+  use_native_modules!
+  use_react_native!(:path => '../node_modules/react-native-macos')
 
   # Enables Flipper.
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable the next line.
-  # use_flipper!()
+  # you should disable these next few lines.
+  use_flipper!
 
+  post_install do |installer|
+    flipper_post_install(installer)
```

Comment:
`react_native_post_install` should be calling `flipper_post_install`. I think this line is redundant.